xl_auth/user/models.py

```diff
@@ -158,6 +159,43 @@ def __repr__(self):
         return '<PasswordReset({email!r})>'.format(email=self.user.email)
 
 
+class LoginAttempt(SurrogatePK, Model):
+    """A failed login attempt."""
+
+    __tablename__ = 'login_attempts'
+    username = Column(db.String(255), unique=False, nullable=False, index=True)
```

Comment:
I think we should drop the indices, for consistency and avoiding clutter with unnecessary *maybe-possibly-good-to-have*'s. We don't have extra indices for anything else in the project and Postgres does a really good job anyway. Let's add these later if we run into delays due to DB having to filter through 1 M rows on login. Or, in that case, maybe we find it's a better approach indexing `created_at`.

Reply:
Removed.